package.json

```diff
@@ -46,12 +46,15 @@
     "ember-native-dom-helpers": "^0.5.4",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.14.0",
-    "loader.js": "^4.2.3"
+    "loader.js": "^4.2.3",
+    "node-sass": "^4.6.0",
+    "node-sass-import-once": "^1.2.0"
   },
   "engines": {
     "node": "^4.5 || 6.* || >= 7.*"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "before": "node-sass-import-once"
```

Comment:
I think this will need to be “before”: “ember-cli-sass”

Reply:
whoops, that makes sense 👍